README.md

```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
 It also includes a few scripts one to create the releases and another to create a blazegraph container.
 
 ./scripts/release/create_release.sh - Requires github username and token.  
-Release process:
-  * List open pull requests
-  * Create the release/$VERSION from develop
-  * Test the releases/$VERSION
-  * From the releases/$VERSION branch, create the pull request to master
-  * Assign and close the pull requests
-  * After the pull request's are approved then create the Tagged Releases
+
+To create a release, the create_release.sh script is run multiple times, between each run there are manual testing and validation steps.
+
+1.  Clone the build repository 
+```
+git clone https://github.com/GRIDAPPSD/gridappsd-docker-build
+```
+
+1.  Clone the gridappsd-docker repository
+```
+git clone https://github.com/GRIDAPPSD/gridapspd-docker
+```
+
+1.  Edit the create_release.sh script to update the version and add GitHub username and token.
+```
+cd gridappsd-docker-build/scripts/release
+vi create_release.sh
+```
+
+1.  Run the create_release.sh script to create the release branches  
+```
+./create_release.sh
+```
+  - pulls the latest gridappsd/blazegraph:develop container and tags as releases_VERSION and pushes to docker hub
+  - clone the GitHub repositories develop branches and create the releases/VERSION branches
+  - Update the default version for the run.sh script in gridappsd-docker
+
+1.  Verify the containers were built and test the releases_VERSION
+```
+cd ../../../gridappsd-docker
+./run.sh -t releases_VERSION
+```
+
+1.  Run the create_release.sh script to create the pull requests
+```
+cd ../gridapspd-docker-build/scripts/release
```

Comment:
so we have create_release.sh in both gridappsd-docker and gridappsd-docker-build?

Reply:
No the script is only in gridappsd-docker-build. I included the gridapspd-docker for the references to the run script for testing.